_pages/clients/python.md

```diff
@@ -64,206 +60,164 @@ client = Algorithmia.client("YOUR_API_KEY", "https://mylocalendpoint")
+Once the algorithm has completed, `response.result` will contain the URI for the resulting file, which you can then download (or provide as input to another algorithm in a pipeline).
 
-### Limits
+### Download the resulting file
 
-Your account can make up to {{site.data.stats.platform.max_num_algo_requests}} Algorithmia requests at the same time (this limit <a onclick="Intercom('show')">can be raised</a> if needed).
+Verify that the file that you want to download exists, and try downloading it to a (new) local file:
 
-## Conclusion
-
-This guide covered installing Algorithmia via pip, uploading and downloading data to and from a user created data collection, checking if a file exists using the Data API, calling an algorithm, and handling errors.
+{% highlight python %}
+# Download the file
+if client.file(response.result).exists() is True:
+    localfile = client.file(response.result).getFile()
+{% endhighlight %}
 
-For more information on the methods available using the Data API in Python check out the [Data API](http://docs.algorithmia.com/?python#data-api-specification) documentation or the [Python Client Docs](https://github.com/algorithmiaio/algorithmia-python).
+This copies the file from your data collection and saves it as a file on your local machine, storing the filename in the variable `localfile`.
 
-For convenience, here is the whole script available to run:
+Alternately, if you just need the binary content of the file to be stored in a variable, you can retrieve the remote file's content without saving the actual file:
 
 {% highlight python %}
-import Algorithmia
-from Algorithmia.acl import ReadAcl, AclType
-
-apiKey = "YOUR_API_KEY"
-# Create the Algorithmia client
-client = Algorithmia.client(apiKey)
+# Download contents of file as a string
+if client.file(response.result).exists() is True:
+    output = client.file(response.result).getBytes()
+{% endhighlight %}
 
-# Set your Data URI
-nlp_directory = client.dir("data://YOUR_USERNAME/nlp_directory")
-# Create your data collection if it does not exist
-if nlp_directory.exists() is False:
-    nlp_directory.create()
+This will get the image as binary data, saving it to the variable `input`, which might be useful when writing algorithms that are part of an image processing pipeline. For more image-manipulation tutorials, see the [Computer Vision Recipes]({{site.baseurl}}/tutorials/recipes/#computer-vision). 
 
-# Create the acl object and check if it's the .my_algos default setting
-acl = nlp_directory.get_permissions()  # Acl object
-acl.read_acl == AclType.my_algos  # True
+If the file was text (an image, etc), you could instead use the function `.getString()` to retrieve the file's content as a string. For more methods on how to get a file using the Data API from a data collection go to the [API Specification](http://docs.algorithmia.com/#getting-a-file).
```

Comment:
nit: can prevent an unnecessary redirect here by using the algorithmia.com domain:
```suggestion
If the file was text (an image, etc), you could instead use the function `.getString()` to retrieve the file's content as a string. For more methods on how to get a file using the Data API from a data collection go to the [API Specification](/developers/api/#get-a-file-or-directory).
```